lib/core/traversalUtils.js

```diff
@@ -41,8 +41,7 @@ export const getICSSExportPropsMap = (ast: gASTNode): classMapType => {
 
 export const getRegularClassesMap = (ast: gASTNode): classMapType => {
   const ruleSets: Array<gASTNode> = [];
-  ast.traverseByType('ruleset', node => ruleSets.push(node));
```

Comment:
Does `ast.traverseByType('class', node => ruleSets.push(node))` not work here?

Reply:
Oh you're right 😅